src/helpers.js

```diff
@@ -91,8 +91,9 @@ module.exports = function(Chart) {
 	function initConfig(config) {
 		config = chartHelpers.configMerge(Chart.Annotation.defaults, config);
 		if (chartHelpers.isArray(config.annotations)) {
-			config.annotations.forEach(function(annotation) {
+			config.annotations = config.annotations.map(function(annotation) {
 				annotation.label = chartHelpers.configMerge(Chart.Annotation.labelDefaults, annotation.label);
+				return chartHelpers.configMerge(Chart.Annotation.annotationDefaults, annotation);
```

Comment:
`configMerge` is being deprecated https://github.com/chartjs/Chart.js/pull/6022. Can you switch these to just use `helpers.merge` instead, which is more appropriate here

Reply:
I've picked up making this change, but switching to `helpers.merge()` causes the annotation not to render, presumably because there is specific logic in `configMerge()` for handling scale.
Is there any guidance / best practice / examples on how to migrate from `configMerge()` to `helpers.merge()`?